src/iOS/NearbyMappersViewController.m

```diff
@@ -6,13 +6,12 @@
 //  Copyright (c) 2012 Bryce Cogswell. All rights reserved.
 //
 
+#import <SafariServices/SafariServices.h>
```

Comment:
This can also be written as

```smalltalk
@import SafariServices;
```

This [Stackoverflow answer][1] gives some insight in the difference between the two.

[1]: https://stackoverflow.com/a/18947634

Reply:
I know about this, but I personally are also very into code consistency. So if this is important we should to this project wide in one PR. Do you agree?

Reply:
Yes, consistency is a good point.
Since the remaining imports are no framework imports, using `@import` here wouldn‘t hurt consistency for _this_ class, IMHO. But I also don‘t think that this is a „must“; that‘s why I wrote „can“. :)

To answer your question: I don‘t think fixing these imports is something we should spend our time on at the moment.